index.js

```diff
@@ -67,7 +67,11 @@ function registerListener(session, options, callback = () => {}) {
 
 		const window_ = majorElectronVersion() >= 12 ? BrowserWindow.fromWebContents(webContents) : getWindowFromWebContents(webContents);
 
-		const directory = options.directory || app.getPath('downloads');
+		let directory = options.directory || app.getPath('downloads');
+		if (directory.match(/^\./)) {
```

Comment:
This is not a good way to handle it. It will fail in so many ways. The correct way is to just `path.resolve` which will passthrough if it's already an absolute path.

Reply:
I was unaware of `path.resolve`. That does seem to work all the time.